WSSiOS/WSSiOS/Source/Data/Entity/Feed/FeedEntity.swift

```diff
@@ -24,10 +24,7 @@ struct FeedEntity {
     
     // 피드 연결 작품 관련
     let hasLinkedNovel: Bool
-    let novelId: Int?
-    let novelTitle: String?
-    let novelRatingCount: Int?
-    let novelRating: Float?
+    let novelData: FeedDetailNovelEntity?
```

Comment:
W2;
```suggestion
    let linkedNovel: FeedDetailNovelEntity?
```
위에 `let hasLinkedNovel: Bool` 변수와 네이밍을 통일시켜서 위처럼 수정해도 좋을 것 같아요!

Reply:
W1;
+) 옵셔널 타입으로 처리하지 않아도 될 것 같아요!
`bindData` 내부에서 `hasLinkedNovel`일 때 UI 바인딩 분기처리를 이미 진행하고 있기 때문에 
현재 변수는 `let linkedNovel: FeedDetailNovelEntity` 로 선언해주면 될 것 같아요!

Reply:
저는 오히려 FeedDetailNovelEntity 내부의 프로퍼티가 모두 옵셔널인게 좀 더 어색하다고 느끼는데,
let novelData: FeedDetailNovelEntity? 이 프로퍼티에서,
연결된 소설이 없으면 그냥 nil을 두면 되고, 연결된 소설이 있다면 작품 제목이나 아이디 등이 모두 있을 것이기 때문에 FeedDetailNovelEntity 내부에서, 연결된 소설이 존재하는 경우에 무조건 있을 수 밖에 없는 프로퍼티는 옵셔널로 처리하지 않는게 더 좋을 것 같아요